x-pack/plugins/endpoint/public/applications/endpoint/view/policy/policy_list.tsx

```diff
@@ -70,6 +72,25 @@ const FormattedDateAndTime: React.FC<{ date: Date }> = ({ date }) => {
   );
 };
 
+const PolicyLink: React.FC<{ name: string; route: string }> = ({ name, route }) => {
+  const history = useHistory();
+
+  return (
+    <EuiLink
+      onClick={(event: React.MouseEvent) => {
```

Comment:
should also add `href` so that user can open in new window/tab or copy url. maybe on next PR :)

Reply:
@paul-tavares I implemented this locally, looks like `EuiLink` doesn't accept both `href` and `onClick`

![image](https://user-images.githubusercontent.com/56395104/76631436-15f1a580-6518-11ea-80d8-a4ff7a412381.png)

We'll have to revisit that, I'll talk with EUI team

Reply:
oh yeah - I remember hitting that recently in ingest. Fortunately, I ended up not needing it there since that project is using hash router.
/cc me on the discussion with EUI if you remember

Reply:
yeah i had to do something like this for the endpoint list, which i put right above the eui component
```  // eslint-disable-next-line @elastic/eui/href-or-on-click ```